game/definitions.rpy

```diff
@@ -1075,6 +1075,15 @@ init python in jn_utils:
         """
         return datetime.datetime.now() - store.jn_globals.current_session_start_time
 
+    def get_total_gameplay_length():
+        """
+        Returns a timedelta object representing the total time the player has spent with Natsuki.
+
+        OUT:
+            datetime.timedelta object representing the length of the total game time
+        """
+        return datetime.datetime.now() - store.persistent.jn_first_visited_date
```

Comment:
For some issues we can potentially have, it may be in our best interest to nonecheck this or have a default value of `datetime.date.today()` as a fallback in case it's somehow None